### ISSUE TITLE
Blade Runner RPG Support

### ISSUE THREAD

**Issue by NocFenix** [Stefouch/sebedius-yearzero-discord-bot#141]
### Who would use it?
Lots of us over at the BR RPG discord server!

### What part of the bot?
- [ ] Dice Roller (Symbols & Specific Mechanics)
- [ ] Add Chases (Foot & Vehicle)
- [ ] Add Critical Injuries (Crushing & Piercing)
- [ ] Add Critical Stress Effects (Human & Replicant)

### How would it work?
It is most similar to T2K, but specific for Blade Runner's rules

### Why should this be added?
This bot supports all of the other Free League games.

**Comment by Stefouch** [comment]
Agree, this will slowly be added to the bot, with first the roll commands

**Comment by Stefouch** [comment]
You can roll dice with `/roll bladerunner` but it will have the dice icons of Twilight 2000.

I'll add the correct dice icons in a near future.

For advantage: use a positive modifier;
For disadvantage: use a negative modifier

**Comment by NocFenix** [comment]
@Stefouch for the advantage/disadvantage, do you mean something like: `/roll bladerunner bc+` because it isn't working for me.

For now, I can just recognize that advantage means add another of my lowest, so in the above scenario it becomes `/roll bladerunner bcc` and disadvantage means remove my lowest, so `/roll bladerunner b`

**Comment by Stefouch** [comment]
Critical Injuries have been added for Blade Runner
